css/stylesheet.css

```diff
@@ -26,3 +26,10 @@
   font-weight: normal;
   font-style: normal;
 }
+
+@font-face {
```

Comment:
- [ ] Add the right style according to the next behavior designs provided by the Fimga design you selected.
- [ ] Add `cursor: pointer;` attribute for all of your buttons in your HTML file when you hover them.

![image](https://user-images.githubusercontent.com/60164239/129861871-bea7f718-b527-46c1-901d-a0aca5201d83.png)